.eslintignore

```diff
@@ -4,4 +4,6 @@
 /examples
 /mithril.js
 /mithril.min.js
+/stream.js
```

Comment:
package.json `main` points to `mithril.js`. So when doing require/import we'd get mithril.js, not index.js, right? So would we not also want to get the bundled, top-level stream.js?

And when using unpkg.com in a jsbin I was thinking it'd be nice to be able to use unpkg.com/mithril/stream rather than unpkg.com/mithril/mithril-stream